app/views/statistics/index.html.erb

```diff
@@ -1,5 +1,3 @@
 <% title _("Statistics"), _('Statistics') %>
 
-<div id="statistics"></div>
-
-<%= mount_react_component('StatisticsChartsList', '#statistics', @metadata.to_json)%>
+<%= mount_react_component('StatisticsChartsList', nil, @metadata.to_json, dom_id: 'statistics-charts-list')%>
```

Comment:
I'm not too happy about using nil in a method signuature, if you design it from scratch, how would write it? perhaps we should add a new method, and have the old mounter call the new method and eventually deprecate it? e.g. like https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/helpers/reactjs_helper.rb ?